react/index.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+import type { FunctionComponent } from 'react'
+
+type Props = {
+  name: string
+}
+
+const Index: FunctionComponent<Props> = ({ name }) => {
```

Comment:
What do you think of avoiding `index.tsx` because of:

```tsx
import { index } from 'vtex.foobar'
```

Reply:
Will call it `Greeting`